### ISSUE TITLE
Skipping thread already processed

### ISSUE THREAD

**Issue by FluffyUnicorn111** [elebumm/RedditVideoMakerBot#151]
Hi there, I'm running the script in a loop and I noticed many times he start processing a thread already done. would be possible to implement a check before starting and skip the thread if already done?

**Comment by FluffyUnicorn111** [comment]
forgot to mention that i'm running it using the "mod" that it create the file with the post name. so it may be easier to check if the same is present maybe in a file list?

**Comment by realdav2** [comment]
hey I wanted to ask if you could send me the bot over to automate it I've got no idea how im supposed to do it haha thx in advance

**Comment by JasonLovesDoggo** [comment]
done
